core/src/main/java/hudson/FilePath.java

```diff
@@ -2045,9 +2031,15 @@ public Void invoke(File f, VirtualChannel channel) throws IOException {
      * Reads this file into a string, by using the current system encoding.
```

Comment:
nit: Would be nice to update the docs here and for `FilePath#write(String, String)` to clarify that _system_ refers to the agent.